src/loadPage.ts

```diff
@@ -19,13 +13,23 @@ export function loadPage({
   // @ts-ignore
   delete global.window;
 
-  // @NOTE Here we have to remove pagePath's trailing "/"
-  const page = require(path.resolve(pagesDirectory, pagePath.substring(1)));
+  require('next/dist/next-server/lib/side-effect');
```

Comment:
Before requiring the whole page, we require only the module that we want to trick to think is on server.